server/controllers/api/assessmentDayDataController/index.js

```diff
@@ -121,6 +131,25 @@ const AssessmentDayDataController = {
         }
       }
 
+      await AssessmentModel.upsert(
+        appDb,
+        assessmentQuery,
+        newAssessmentProperties
+      )
+
+      if (assessment_variables.length) {
+        await Promise.all(
+          assessment_variables.map(
+            async (variableMetadata) =>
```

Comment:
what is the shape of the variable metadata? How are we associating the assessment variable with the assessment?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/726#discussion_r1559489677